server.js

```diff
@@ -0,0 +1,49 @@
+var express = require("express");
+var app = express();
+var port = process.env.PORT || 3000;
+var bodyparser = require("body-parser");
+var piglatinify = require("./lib/piglatinify.js");
+var rL = require("./lib/randomLetters.js");
```

Comment:
this works, but better to get in the habit of using longer variable names - e.g. quote, random, joke, randomLetters. If I were collaborating with you on this code, this would make it really hard for me to read it.